packages/clay-css/src/scss/mixins/_buttons.scss

```diff
@@ -569,13 +572,14 @@
 			$mobile-c-inner
 		);
 
-		$unset: setter(
+		$unset: if(
+			variable-exists(clay-unset-placeholder),
+			$clay-unset-placeholder,
 			if(
-				variable-exists(clay-unset-placeholder),
-				$clay-unset-placeholder,
-				$cadmin-clay-unset-placeholder
-			),
-			clay-unset-placeholder
+				variable-exists(cadmin-clay-unset-placeholer),
+				$cadmin-clay-unset-placeholer,
+				clay-unset-placeholder
```

Comment:
@matuzalemsteles It wouldn't fail because it would default to `clay-unset-placeholder`. Sass would try to `@extend clay-unset-placeholder !optional` that doesn't exist. The [optional flag](https://sass-lang.com/documentation/at-rules/extend#mandatory-and-optional-extends) tells Sass to do nothing if the selector doesn't exist. I could have chosen a better name for it though.

Reply:
Yeah I got it but the issue I'm seeing here is that before we are checking if `variable-exists(clay-unset-placeholder)` exists, otherwise we are checking if `variable-exists(cadmin-clay-unset-placeholer)` exists if not, we're re-adding `clay-unset-placeholder`, I believe this would fail because we've checked before that it doesn't exist, shouldn't this be a hardcoded value here like the other use cases of this PR?

Reply:
In Clay, one will always exist, but this will still work if both don't exist. the `variable-exists()` function always returns a boolean value. The if statements are like nested ternaries in javascript. With that said, I agree we should just hard code `clay-unset-placeholder`. I just converted and didn't think about it.